packages/votura-validators/src/objects/response.ts

```diff
@@ -1,7 +1,15 @@
 import { z, type ZodError } from 'zod/v4';
 import { voturaMetadataRegistry } from '../voturaMetadateRegistry.js';
 
+export const responseObject = z.object({
```

Comment:
Maybe rename this to `response4xxObject`? Or should this be used for anything else than error messages?
```suggestion
export const response4xxObject = z.object({
```